### PR TITLE
Verify node points to right tuned profile

### PR DESCRIPTION
**Pull request by Avielyo10**

### CODE REVIEW

**Comment by cynepco3hahue** [comment]
/approve

@yanirq Can you please review it?

File: functests/1_performance/performance.go [hunk 1]
```diff
@@ -116,6 +116,15 @@ var _ = Describe("[rfe_id:27368][performance]", func() {
 			}, 120*time.Second, testPollInterval*time.Second).Should(BeTrue(),
 				"tuned CR name owned by a performance profile CR should only be "+tunedExpectedName)
 		})
+
+		It("Node should point to right tuned profile", func() {
```
Comment (gsr-shanks) [inline functests/1_performance/performance.go:120]: Please add a test_id

Comment (Avielyo10) [reply functests/1_performance/performance.go]: Done

**Comment by gsr-shanks** [comment]
LGTM 👍

**Pull request merged**